server/pull_request.go

```diff
@@ -30,7 +30,12 @@ func (s *Server) handlePullRequestEvent(ctx context.Context, event *PullRequestE
 		if err := s.handleCheckCLA(ctx, pr); err != nil {
 			mlog.Error("Unable to check CLA", mlog.Err(err))
 		}
-		s.triggerCircleCiIfNeeded(ctx, pr)
+
+		if err := s.triggerCircleCIIfNeeded(ctx, pr); err != nil {
+			mlog.Error("Unable to trigger CircleCI", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number), mlog.String("fullname", pr.FullName), mlog.Err(err))
+		}
+		mlog.Debug("Triggered CircleCI", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number), mlog.String("fullname", pr.FullName))
```

Comment:
This needs to be in an else condition. Same applies to the 2 instances below.

Reply:
🤦